manifests/install.pp

```diff
@@ -20,12 +21,12 @@
   String $path                           = $letsencrypt::path,
   String $repo                           = $letsencrypt::repo,
   String $version                        = $letsencrypt::version,
+  Array[String] $vcs_dependencies        = $letsencrypt::vcs_dependencies,
```

Comment:
I think  empty are also not here allowed / don't make sense?
```suggestion
  Array[String[1]] $vcs_dependencies     = $letsencrypt::vcs_dependencies,
```

Reply:
Updated